pkg/servicemesh/controller/memberroll/controller.go

```diff
@@ -117,7 +117,7 @@ func (smmrc *serviceMeshMemberRollController) Start(stop chan struct{}) {
 	for _, obj := range smmrc.informer.GetStore().List() {
 		serviceMeshMemberRoll := getServiceMeshMemberRoll(obj)
 		if smmrc.memberRollName == serviceMeshMemberRoll.Name {
-			seedNamespaces := smmrc.getNamespaces(serviceMeshMemberRoll.Status.ConfiguredMembers)
```

Comment:
does `.ConfiguredMembers` now contain the control plane namespace? if not, this is changing behavior, right?

Reply:
The control plane namespace gets added in https://github.com/Maistra/istio/blob/e25b1b454352642d6b52fd241344d6c00a1ae71a/pkg/servicemesh/controller/memberroll/controller.go#L216

Reply:
yes, but that isn't called for seeding anymore now, right? maybe I'm overlooking something

Reply:
oh nevermind it is called in the updateNamespaces func. ignore my comment